apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -295,11 +303,7 @@ public static Instant parseTimestamp(String timestampText) {
    *     Optional if not
    */
   public static Optional<Instant> parseOptionalTimestamp(String timestampText) {
-    if (timestampText.isEmpty()) {
-      return Optional.empty();
-    } else {
-      return Optional.of(parseTimestamp(timestampText));
-    }
+    return timestampText.isEmpty() ? Optional.empty() : Optional.of(parseTimestamp(timestampText));
```

Comment:
I see a lot of these changes in here -- I think we ought to add this to a future style guide discussion before changing them everywhere. It's a personal preference that we should discuss once and adopt a standard for. Without that I think changing them everywhere is premature.

Reply:
I 'sort of' agree.....it goes back to my firm belief of brackets & scoping (and another one of multiple return statements). So I could change the code to include the brackets which will result in 5 ilnes of change:

if (timestampText.isEmpty()) {
    return Optional.empty();
} else {
    return Optional.of(parseTimestamp(timestampText));
}

Now as a lazy developer, I'd rather have one line than 5!

Reply:
agreed; however I did have to introduce code (new parser for bigint, smallint, and integer); and I will always use the ternary operator for these simple if-then-else. So my conundrum was add my changes using ternary and then field questions as to why didn't I use a verbose if-then-else.

Reply:
IMO ternaries are fine for simple logic splits like this, but is certainly a good thing to put on the list of things to discuss as part of a greater style guide.